plugins/modules/k8s_scale.py

```diff
@@ -82,6 +96,15 @@
     resource_definition: "{{ lookup('file', '/myproject/elastic_deployment.yml') | from_yaml }}"
     replicas: 3
     wait: no
+
+- name: Scale deployment using label selectors (continue operation in case error occured on one resource)
+  kubernetes.core.k8s_scale:
+    replicas: 3
+    kind: Deployment
+    namespace: test
+    label_selectors:
+      - app = test
```

Comment:
Since this is string I would avoid spaces in between 

```suggestion
      - app=test
```